src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -488,7 +496,12 @@ internal bool IsIntegrationEnabled(string integrationName)
 
         internal bool IsNetStandardFeatureFlagEnabled()
         {
-            var value = EnvironmentHelpers.GetEnvironmentVariable("DD_TRACE_NETSTANDARD_ENABLED", string.Empty);
+            return IsFeatureFlagEnabled(ConfigurationKeys.FeatureFlags.NetStandardEnabled);
+        }
+
+        internal bool IsFeatureFlagEnabled(string featureFlag)
+        {
+            var value = EnvironmentHelpers.GetEnvironmentVariable(featureFlag, string.Empty);
```

Comment:
For all boolean settings, we should use the `string.ToBoolean()` extension method:
https://github.com/DataDog/dd-trace-dotnet/blob/d9f44e4ff727dcd6693b887fcc05a7fbd2eb70fc/src/Datadog.Trace/ExtensionMethods/StringExtensions.cs#L30

`DD_TRACE_NETSTANDARD_ENABLED` is a special case. The setting is only used in native code. `IsNetStandardFeatureFlagEnabled()` only checks for `"1"` and `"true"` to match the logic in C++ and it is only used for logging purposes:
https://github.com/DataDog/dd-trace-dotnet/blob/d9f44e4ff727dcd6693b887fcc05a7fbd2eb70fc/src/Datadog.Trace/Tracer.cs#L538

Reply:
> `DD_TRACE_NETSTANDARD_ENABLED` is a special case. The setting is only used in native code.

Ah, good to know, thanks! Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1288/commits/4b0e7b1ac49093fe57bdd001c90d2bb783dca545